source/install/install-rhel-66.rst

```diff
@@ -0,0 +1,23 @@
+.. _install-rhel-66:
+
+=================================
+Installing Mattermost on RHEL 6.6
+=================================
+
+Install a production-ready Mattermost system on 1 to 3 machines.
+
+.. include:: install-common-intro.rst
+
+.. contents:: Install and configure the components in the following order. Note that you need only one database, either MySQL or PostgreSQL.
+  :backlinks: top
+  :local:
+  
+.. include:: install-rhel-66-server.rst
```

Comment:
I like the use of includes for docs that are re-used in multiple docs, but what's the purpose of doing includes for the docs that will only appear in a single doc? ie: `.. include:: install-rhel-66-mysql.rst` , `install-rhel-66-postgresql.rst`, etc